nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/shape/Linspace.java

```diff
@@ -46,6 +46,11 @@ public Linspace(SameDiff sameDiff, SDVariable from, SDVariable to, SDVariable le
         this.dataType = dataType;
     }
 
+    public Linspace(DataType dataType, double start, double stop, double number){
+        //TODO: implement this constructor for NDBase.java.
+        this.dataType = dataType;
```

Comment:
https://github.com/eclipse/deeplearning4j/issues/8513

For now: let's do: `super(new INDArray[]{Nd4j.scalar(start), Nd4j.scalar(stop), Nd4j.scalar(number)}, null)` and switch to targs once linked issue is done.